src/pytestqt/modeltest.py

```diff
@@ -461,11 +461,11 @@ def _test_data(self):
             (qt_api.QtCore.Qt.SizeHintRole, qt_api.QtCore.QSize),
             (qt_api.QtCore.Qt.FontRole, qt_api.QtGui.QFont),
             (
-                qt_api.QtCore.Qt.BackgroundColorRole,
+                qt_api.QtCore.Qt.BackgroundRole,
                 (qt_api.QtGui.QColor, qt_api.QtGui.QBrush),
             ),
             (
-                qt_api.QtCore.Qt.TextColorRole,
+                qt_api.QtCore.Qt.ForegroundRole,
```

Comment:
I wondered if we should provide backwards-compatibility there, but it looks like Qt's own model tester did the same anyways: https://code.qt.io/cgit/qt/qtbase.git/commit/src/testlib/qabstractitemmodeltester.h?id=e9bebc12812b5c50346952cd9128bdd08ddd0b9d

So I guess it's fine.

Reply:
They do have the same values in Qt 5.  https://doc.qt.io/qt-5/qt.html#ItemDataRole-enum

Reply:
Yeah, but I was wondering when the new names were introduced - the old ones were deprecated in Qt 5.13, but it looks like the new ones have been around since 4.2 according to https://codereview.qt-project.org/c/qt/qtbase/+/243160. So yup, this certainly isn't an issue.